sw/device/tests/power_virus_systemtest.c

```diff
@@ -1150,6 +1154,44 @@ static void max_power_task(void *task_parameters) {
     CHECK_ARRAYS_EQ(received_uart_data, kUartMessage, num_uart_rx_bytes);
   }
 
+  // Check I2C transactions. Only for DVsim
+  if (kDeviceType == kDeviceSimDV) {
+    uint8_t tx_fifo_lvl, rx_fifo_lvl;
+
+    // Make sure all fifo entries have been drained.
+    for (uint32_t ii = 0; ii < ARRAYSIZE(i2c_handles); ++ii) {
+      do {
+        CHECK_DIF_OK(dif_i2c_get_fifo_levels(i2c_handles[ii], &tx_fifo_lvl,
+                                             &rx_fifo_lvl, NULL, NULL));
```

Comment:
since you are only checking the tx FIFO level, I think the pointer to `rx_fifo_level` can be `NULL`: https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/lib/dif/dif_i2c.h;drc=b41d05e93b15a8911838828f35f99c885e564e4b;l=547

Reply:
thanks Tim :) Done.